config/config.go

```diff
@@ -26,6 +26,9 @@ type SuperKeyWorkerConfig struct {
 	SourcesScheme      string
 	SourcesPort        int
 	SourcesPSK         string
+	OIDCIssuer         string
+	OIDCClientID       string
+	OIDCClientSecret   string
```

Comment:
We probably need a configurable timeout.  We might also need the scope(s) to be configurable (maybe?).